eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
- [ ] You have added the file as requested previously, however, you are missing the dot `.` before the filename. Kindly rename your file to `.eslintrc.json`. This way Github Actions will run your ESLint check. Ensure that both Stylelint and ESLint are being run on the Pull Request.